mautic/mautic
# Hide twitter related item if disabled

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | No
| New feature/enhancement? (use the a.x branch)      | Yes
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | Yes 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Hide following twitter related item if twitter is disabled :-

1. Social monitoring in Channel menu
2. Social tab in contact detail page
3. Social setting in global configuration
4. Social login field in form fields



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. verify following should be hidden if twitter is disabled :-

> 1. Social monitoring in Channel menu
> 2. Social tab in contact detail page
> 3. Social setting in global configuration
> 4. Social login field in form fields
